LT Serbian Language

strn opened the pull request:
Pull request for merging LT RPM module.

strn commented:
Pull request for merging LT RPM module.

strn commented:
Reopened pull request

danielnaber commented:
Could you check why travis says the build fails?

janschreiber commented:
```
[INFO] Scanning for projects...

[ERROR] [ERROR] Some problems were encountered while processing the POMs:

[FATAL] Non-readable POM /home/travis/build/languagetool-org/languagetool/pom.xml: no more data available - expected end tag </project> to close start tag <project> from line 1, parser stopped on END_TAG seen ...</modules>\n</project>\n... @365:1 @ 
```

strn commented:
Hi Daniel,


I've checked the LT build, and it looks fine to me when I build it 
locally. POM is OK from syntactical point of view.

What kind of special processing does Travis do to the POM?


Regards,
Zoltan

Дана 2017-08-17 20:32, Daniel Naber написа:
> Could you check why travis says the build fails?
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/languagetool-org/languagetool/pull/769#issuecomment-323156859
> [2]
> https://github.com/notifications/unsubscribe-auth/AJE_UvoT3plMe2_Q_7b6829TC8Qio6W3ks5sZIdHgaJpZM4O1g4q

strn commented:
Hi Daniel,


Attached is POM file as it is on my system. Can you please try replacing 
POM on your side and rebuilding it?


Regards,
Zoltan


Дана 2017-08-17 20:32, Daniel Naber написа:
> Could you check why travis says the build fails?
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/languagetool-org/languagetool/pull/769#issuecomment-323156859
> [2]
> https://github.com/notifications/unsubscribe-auth/AJE_UvoT3plMe2_Q_7b6829TC8Qio6W3ks5sZIdHgaJpZM4O1g4q

<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>

	<prerequisites>
		<maven>3.04</maven>
	</prerequisites>

	<parent>
		<groupId>org.sonatype.oss</groupId>
		<artifactId>oss-parent</artifactId>
		<version>7</version>
	</parent>

	<scm>
		<connection>scm:git:git@github.com:languagetool-org/languagetool.git</connection>
		<developerConnection>scm:git:git@github.com:languagetool-org/languagetool.git</developerConnection>
		<url>git@github.com:languagetool-org/languagetool.git</url>
		<tag>HEAD</tag>
	</scm>

	<groupId>org.languagetool</groupId>
	<artifactId>languagetool-parent</artifactId>
	<version>3.9-SNAPSHOT</version>
	<packaging>pom</packaging>

	<properties>
		<languagetool.version>3.9-SNAPSHOT</languagetool.version>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
		<maven.compiler.source>1.8</maven.compiler.source>
		<maven.compiler.target>1.8</maven.compiler.target>
		<maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
		<maven.jar.plugin>2.6</maven.jar.plugin>  <!-- NOTE: don't update without testing OpenOffice, 3.0.2 caused "Got no 
			data stream!" after add-on installation -->
		<junit.version>4.12</junit.version>
		<morfologik.version>2.1.3</morfologik.version>
		<jackson.version>2.8.4</jackson.version>
		<lucene.version>5.5.3</lucene.version>
	</properties>

	<build>

		<extensions>
			<!-- needed to deploy to https://repository-languagetool.forge.cloudbees.com/snapshot/ -->
			<extension>
				<groupId>org.apache.maven.wagon</groupId>
				<artifactId>wagon-webdav</artifactId>
				<version>1.0-beta-2</version>
			</extension>
		</extensions>

		<pluginManagement>
			<plugins>

				<plugin>
					<artifactId>maven-compiler-plugin</artifactId>
					<version>3.6.1</version>
				</plugin>

				<plugin>
					<artifactId>maven-surefire-plugin</artifactId>
					<version>2.20</version>
					<configuration>
						<trimStackTrace>false</trimStackTrace>
						<argLine>-Xms256m -Xmx950m</argLine>
						<runOrder>failedfirst</runOrder>
						<excludes>
							<exclude>**/*ConcurrencyTest.java</exclude>
							<exclude>**/Abstract*.java</exclude>
						</excludes>
					</configuration>
				</plugin>

				<plugin>
					<groupId>org.apache.maven.plugins</groupId>
					<artifactId>maven-javadoc-plugin</artifactId>
					<version>2.10.4</version>
					<configuration>
						<bottom><![CDATA[&nbsp;]]></bottom>
						<excludePackageNames>org.languagetool.dev.conversion*</excludePackageNames>
					</configuration>
				</plugin>

				<plugin>
					<groupId>de.dentrassi.maven</groupId>
					<artifactId>rpm</artifactId>
					<version>0.9.2</version>
				</plugin>
			</plugins>
		</pluginManagement>
	</build>

	<distributionManagement>
		<repository>
			<id>community-release</id>
			<url>dav:https://repository-languagetool.forge.cloudbees.com/release/</url>
		</repository>
		<snapshotRepository>
			<id>community-snapshot</id>
			<url>dav:https://repository-languagetool.forge.cloudbees.com/snapshot/</url>
		</snapshotRepository>
	</distributionManagement>

	<profiles>
		<!-- taken from https://docs.sonatype.org/display/Repository/How+To+Generate+PGP+Signatures+With+Maven -->
		<profile>
			<id>release-sign-artifacts</id>
			<activation>
				<property>
					<name>performRelease</name>
					<value>true</value>
				</property>
			</activation>
			<build>
				<plugins>
					<plugin>
						<groupId>org.apache.maven.plugins</groupId>
						<artifactId>maven-gpg-plugin</artifactId>
						<version>1.6</version>
						<configuration>
							<useAgent>true</useAgent>
						</configuration>
						<executions>
							<execution>
								<id>sign-artifacts</id>
								<phase>verify</phase>
								<goals>
									<goal>sign</goal>
								</goals>
							</execution>
						</executions>
					</plugin>
				</plugins>
			</build>
		</profile>
	</profiles>

	<modules>
		<module>languagetool-core</module>
		<module>languagetool-language-modules/en</module>
		<module>languagetool-language-modules/fa</module>
		<module>languagetool-language-modules/fr</module>
		<module>languagetool-language-modules/de</module>
		<module>languagetool-language-modules/pl</module>
		<module>languagetool-language-modules/ca</module>
		<module>languagetool-language-modules/it</module>
		<module>languagetool-language-modules/br</module>
		<module>languagetool-language-modules/nl</module>
		<module>languagetool-language-modules/pt</module>
		<module>languagetool-language-modules/ru</module>
		<module>languagetool-language-modules/ast</module>
		<module>languagetool-language-modules/be</module>
		<module>languagetool-language-modules/zh</module>
		<module>languagetool-language-modules/da</module>
		<module>languagetool-language-modules/eo</module>
		<module>languagetool-language-modules/gl</module>
		<module>languagetool-language-modules/el</module>
		<module>languagetool-language-modules/is</module>
		<module>languagetool-language-modules/ja</module>
		<module>languagetool-language-modules/km</module>
		<module>languagetool-language-modules/lt</module>
		<module>languagetool-language-modules/ml</module>
		<module>languagetool-language-modules/ro</module>
		<module>languagetool-language-modules/sk</module>
		<module>languagetool-language-modules/sl</module>
		<module>languagetool-language-modules/es</module>
		<module>languagetool-language-modules/sv</module>
		<module>languagetool-language-modules/ta</module>
		<module>languagetool-language-modules/tl</module>
		<module>languagetool-language-modules/uk</module>
		<module>languagetool-language-modules/de-DE-x-simple-language</module>
		<module>languagetool-language-modules/all</module>
		<module>languagetool-gui-commons</module>
		<module>languagetool-commandline</module>
		<module>languagetool-standalone</module>
		<module>languagetool-office-extension</module>
		<module>hunspell-native-libs</module>
		<module>languagetool-wikipedia</module>
		<module>languagetool-server</module>
		<module>languagetool-http-client</module>
		<module>languagetool-tools</module>
		<module>languagetool-dev</module>
		<!-- don't add languagetool-client-example here, it's built manually only -->
		<!-- RPM module for building LanguageTool RPMs -->
		<module>languagetool-rpm-package</module>
	</modules>

</project>

danielnaber commented on pom.xml:
```diff
@@ -1,3 +1,189 @@
+		<!-- don't add languagetool-client-example here, it's built manually only -->
+		<module>languagetool-rpm-package</module>
+	</modules>
+=======
```
This looks like a syntax error. The fact that there's so many changes shown here against pom.xml seems to indicate some issue with the way the PR was created, maybe?

strn replied:
I created "pull" request from github.com, using its functionality. Just followed steps, nothing special.

danielnaber replied:
Then could you just email me a ZIP file with all your changes?

strn commented:
Hi Daniel,


Sure, attached are the entire LT RPM module (place it along other LT 
modules), plus top-level pom.xml file.
Please let me know what I did wrong, so that I can avoid the same 
mistake in the future.

Regards,
Zoltan


Дана 2017-08-17 22:02, Daniel Naber написа:
> @DANIELNABER commented on this pull request.
> -------------------------
> 
> In pom.xml [1]:
> 
>> +
> <module>languagetool-language-modules/de-DE-x-simple-language</module>
> +        <module>languagetool-language-modules/all</module>
> +        <module>languagetool-gui-commons</module>
> +        <module>languagetool-commandline</module>
> +        <module>languagetool-standalone</module>
> +        <module>languagetool-office-extension</module>
> +        <module>hunspell-native-libs</module>
> +        <module>languagetool-wikipedia</module>
> +        <module>languagetool-server</module>
> +        <module>languagetool-http-client</module>
> +        <module>languagetool-tools</module>
> +        <module>languagetool-dev</module>
> +        <!-- don't add languagetool-client-example here, it's built
> manually only -->
> +        <module>languagetool-rpm-package</module>
> +    </modules>
> +=======
> 
> Then could you just email me a ZIP file with all your changes?
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/languagetool-org/languagetool/pull/769#discussion_r133814078
> [2]
> https://github.com/notifications/unsubscribe-auth/AJE_UoFeuR_WYHR5O1uTOjT3J9qVplfJks5sZJw7gaJpZM4O1g4q

<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>

	<prerequisites>
		<maven>3.04</maven>
	</prerequisites>

	<parent>
		<groupId>org.sonatype.oss</groupId>
		<artifactId>oss-parent</artifactId>
		<version>7</version>
	</parent>

	<scm>
		<connection>scm:git:git@github.com:languagetool-org/languagetool.git</connection>
		<developerConnection>scm:git:git@github.com:languagetool-org/languagetool.git</developerConnection>
		<url>git@github.com:languagetool-org/languagetool.git</url>
		<tag>HEAD</tag>
	</scm>

	<groupId>org.languagetool</groupId>
	<artifactId>languagetool-parent</artifactId>
	<version>3.9-SNAPSHOT</version>
	<packaging>pom</packaging>

	<properties>
		<languagetool.version>3.9-SNAPSHOT</languagetool.version>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
		<maven.compiler.source>1.8</maven.compiler.source>
		<maven.compiler.target>1.8</maven.compiler.target>
		<maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
		<maven.jar.plugin>2.6</maven.jar.plugin>  <!-- NOTE: don't update without testing OpenOffice, 3.0.2 caused "Got no 
			data stream!" after add-on installation -->
		<junit.version>4.12</junit.version>
		<morfologik.version>2.1.3</morfologik.version>
		<jackson.version>2.8.4</jackson.version>
		<lucene.version>5.5.3</lucene.version>
	</properties>

	<build>

		<extensions>
			<!-- needed to deploy to https://repository-languagetool.forge.cloudbees.com/snapshot/ -->
			<extension>
				<groupId>org.apache.maven.wagon</groupId>
				<artifactId>wagon-webdav</artifactId>
				<version>1.0-beta-2</version>
			</extension>
		</extensions>

		<pluginManagement>
			<plugins>

				<plugin>
					<artifactId>maven-compiler-plugin</artifactId>
					<version>3.6.1</version>
				</plugin>

				<plugin>
					<artifactId>maven-surefire-plugin</artifactId>
					<version>2.20</version>
					<configuration>
						<trimStackTrace>false</trimStackTrace>
						<argLine>-Xms256m -Xmx950m</argLine>
						<runOrder>failedfirst</runOrder>
						<excludes>
							<exclude>**/*ConcurrencyTest.java</exclude>
							<exclude>**/Abstract*.java</exclude>
						</excludes>
					</configuration>
				</plugin>

				<plugin>
					<groupId>org.apache.maven.plugins</groupId>
					<artifactId>maven-javadoc-plugin</artifactId>
					<version>2.10.4</version>
					<configuration>
						<bottom><![CDATA[&nbsp;]]></bottom>
						<excludePackageNames>org.languagetool.dev.conversion*</excludePackageNames>
					</configuration>
				</plugin>

				<plugin>
					<groupId>de.dentrassi.maven</groupId>
					<artifactId>rpm</artifactId>
					<version>0.9.2</version>
				</plugin>
			</plugins>
		</pluginManagement>
	</build>

	<distributionManagement>
		<repository>
			<id>community-release</id>
			<url>dav:https://repository-languagetool.forge.cloudbees.com/release/</url>
		</repository>
		<snapshotRepository>
			<id>community-snapshot</id>
			<url>dav:https://repository-languagetool.forge.cloudbees.com/snapshot/</url>
		</snapshotRepository>
	</distributionManagement>

	<profiles>
		<!-- taken from https://docs.sonatype.org/display/Repository/How+To+Generate+PGP+Signatures+With+Maven -->
		<profile>
			<id>release-sign-artifacts</id>
			<activation>
				<property>
					<name>performRelease</name>
					<value>true</value>
				</property>
			</activation>
			<build>
				<plugins>
					<plugin>
						<groupId>org.apache.maven.plugins</groupId>
						<artifactId>maven-gpg-plugin</artifactId>
						<version>1.6</version>
						<configuration>
							<useAgent>true</useAgent>
						</configuration>
						<executions>
							<execution>
								<id>sign-artifacts</id>
								<phase>verify</phase>
								<goals>
									<goal>sign</goal>
								</goals>
							</execution>
						</executions>
					</plugin>
				</plugins>
			</build>
		</profile>
	</profiles>

	<modules>
		<module>languagetool-core</module>
		<module>languagetool-language-modules/en</module>
		<module>languagetool-language-modules/fa</module>
		<module>languagetool-language-modules/fr</module>
		<module>languagetool-language-modules/de</module>
		<module>languagetool-language-modules/pl</module>
		<module>languagetool-language-modules/ca</module>
		<module>languagetool-language-modules/it</module>
		<module>languagetool-language-modules/br</module>
		<module>languagetool-language-modules/nl</module>
		<module>languagetool-language-modules/pt</module>
		<module>languagetool-language-modules/ru</module>
		<module>languagetool-language-modules/ast</module>
		<module>languagetool-language-modules/be</module>
		<module>languagetool-language-modules/zh</module>
		<module>languagetool-language-modules/da</module>
		<module>languagetool-language-modules/eo</module>
		<module>languagetool-language-modules/gl</module>
		<module>languagetool-language-modules/el</module>
		<module>languagetool-language-modules/is</module>
		<module>languagetool-language-modules/ja</module>
		<module>languagetool-language-modules/km</module>
		<module>languagetool-language-modules/lt</module>
		<module>languagetool-language-modules/ml</module>
		<module>languagetool-language-modules/ro</module>
		<module>languagetool-language-modules/sk</module>
		<module>languagetool-language-modules/sl</module>
		<module>languagetool-language-modules/es</module>
		<module>languagetool-language-modules/sv</module>
		<module>languagetool-language-modules/ta</module>
		<module>languagetool-language-modules/tl</module>
		<module>languagetool-language-modules/uk</module>
		<module>languagetool-language-modules/de-DE-x-simple-language</module>
		<module>languagetool-language-modules/all</module>
		<module>languagetool-gui-commons</module>
		<module>languagetool-commandline</module>
		<module>languagetool-standalone</module>
		<module>languagetool-office-extension</module>
		<module>hunspell-native-libs</module>
		<module>languagetool-wikipedia</module>
		<module>languagetool-server</module>
		<module>languagetool-http-client</module>
		<module>languagetool-tools</module>
		<module>languagetool-dev</module>
		<!-- don't add languagetool-client-example here, it's built manually only -->
		<!-- RPM module for building LanguageTool RPMs -->
		<module>languagetool-rpm-package</module>
	</modules>

</project>

danielnaber commented:
I think attachments don't work when replying via email to github, can you send the ZIP directly via email?

strn commented:
Hi Daniel,


Attached is ZIP archive. If you did not get it, please let me know what 
other email address I shall use.


Regards,
Zoltan

Дана 2017-08-18 10:02, Daniel Naber написа:
> I think attachments don't work when replying via email to github, can
> you send the ZIP directly via email?
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/languagetool-org/languagetool/pull/769#issuecomment-323287785
> [2]
> https://github.com/notifications/unsubscribe-auth/AJE_UqbPzRuvvDmI2GH-StGvz_z04kcFks5sZUUrgaJpZM4O1g4q

danielnaber commented:
I didn't get it, please email it to naber .att. danielnaber dot de

danielnaber commented:
Thanks, I have now added this manually. I replaced a few occurrences of `Language Tool` by `LanguageTool`. Could you add your name and contact information to the `README`?

strn commented:
hi Daniel,

Yes, I can add my contact info to the README. However, which README are 
you referring to:

- My LT fork, or
- ZIP archive I sent you?

Regards,
Zoltan

P.S. What was the problem with pull request? What did I do wrong?



Дана 2017-08-18 15:34, Daniel Naber написа:
> Thanks, I have now added this manually. I replaced a few occurrences
> of Language Tool by LanguageTool. Could you add your name and contact
> information to the README?
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> 	*
> 
> Links:
> ------
> [1]
> https://github.com/languagetool-org/languagetool/pull/769#issuecomment-323354913
> [2]
> https://github.com/notifications/unsubscribe-auth/AJE_UhfI8H1CTn1F8fOQlzk7fnM2BCpwks5sZZLLgaJpZM4O1g4q

danielnaber commented:
I was referring to the ZIP you send. But now that it has been committed, you can just create pull requests against the latest LT. I don't know what was wrong with your original pull request.
